TasksTests are failing

```py
======================================================================
FAIL: test_adds_new_nodes_and_populates_defaults (tests.tasks_tests.UpdateResourceInstanceDataTaskTests.test_adds_new_nodes_and_populates_defaults)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/unittest/mock.py", line 1424, in patched
    return func(*newargs, **newkeywargs)
  File "/Users/jwalls/prj/arches/tests/tasks_tests.py", line 342, in test_adds_new_nodes_and_populates_defaults
    self.assertEqual(
    ~~~~~~~~~~~~~~~~^
        tile.data,
        ^^^^^^^^^^
    ...<4 lines>...
        },
        ^^
    )
    ^
AssertionError: {'99d[71 chars]ection': 'ltr'}}, '6f19f457-7460-409d-8de8-08ef2aa4e731': None} != {'99d[71 chars]ection': 'ltr'}}}
- {'6f19f457-7460-409d-8de8-08ef2aa4e731': None,
-  '99db1e9f-1416-4f35-b06d-54335cf4a3b4': {'en': {'direction': 'ltr',
? ^

+ {'99db1e9f-1416-4f35-b06d-54335cf4a3b4': {'en': {'direction': 'ltr',
? ^

                                                   'value': 'test value'}}}

======================================================================
FAIL: test_deletes_tiles_when_nodegroup_is_deleted (tests.tasks_tests.UpdateResourceInstanceDataTaskTests.test_deletes_tiles_when_nodegroup_is_deleted)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/unittest/mock.py", line 1424, in patched
    return func(*newargs, **newkeywargs)
  File "/Users/jwalls/prj/arches/tests/tasks_tests.py", line 528, in test_deletes_tiles_when_nodegroup_is_deleted
    self.assertEqual(
    ~~~~~~~~~~~~~~~~^
        tile.data,
        ^^^^^^^^^^
    ...<4 lines>...
        },
        ^^
    )
    ^
AssertionError: {'99d[74 chars]ection': 'ltr'}}, '6f19f457-7460-409d-8de8-08ef2aa4e731': None} != {'99d[74 chars]ection': 'ltr'}}}
- {'6f19f457-7460-409d-8de8-08ef2aa4e731': None,
-  '99db1e9f-1416-4f35-b06d-54335cf4a3b4': {'en': {'direction': 'ltr',
? ^

+ {'99db1e9f-1416-4f35-b06d-54335cf4a3b4': {'en': {'direction': 'ltr',
? ^

                                                   'value': 'Default Value'}}}

======================================================================
FAIL: test_updates_defaults (tests.tasks_tests.UpdateResourceInstanceDataTaskTests.test_updates_defaults)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/unittest/mock.py", line 1424, in patched
    return func(*newargs, **newkeywargs)
  File "/Users/jwalls/prj/arches/tests/tasks_tests.py", line 459, in test_updates_defaults
    self.assertEqual(
    ~~~~~~~~~~~~~~~~^
        tile.data,
        ^^^^^^^^^^
    ...<4 lines>...
        },
        ^^
    )
    ^
AssertionError: {'99d[74 chars]ection': 'ltr'}}, '6f19f457-7460-409d-8de8-08ef2aa4e731': None} != {'99d[74 chars]ection': 'ltr'}}}
- {'6f19f457-7460-409d-8de8-08ef2aa4e731': None,
-  '99db1e9f-1416-4f35-b06d-54335cf4a3b4': {'en': {'direction': 'ltr',
? ^

+ {'99db1e9f-1416-4f35-b06d-54335cf4a3b4': {'en': {'direction': 'ltr',
? ^

                                                   'value': 'Default Value'}}}

----------------------------------------------------------------------
Ran 7 tests in 2.380s

FAILED (failures=3)
```

--- a/tests/tasks_tests.py
+++ b/tests/tasks_tests.py
@@ -339,13 +339,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             sortorder=0,
         )
 
-        self.assertEqual(
-            tile.data,
-            {
-                str(self.string_node_id): {
-                    "en": {"value": "test value", "direction": "ltr"},
-                }
-            },
+        self.assertDictEqual(
+            tile.data[str(self.string_node_id)],
+            {"en": {"value": "test value", "direction": "ltr"}},
         )
 
         draft_graph = self.test_graph.create_draft_graph()
@@ -400,11 +396,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
 
         tile.refresh_from_db()
 
-        self.assertEqual(
+        self.assertDictEqual(
             tile.data[str(self.string_node_id)],
-            {
-                "en": {"value": "test value", "direction": "ltr"},
-            },
+            {"en": {"value": "test value", "direction": "ltr"}},
         )
 
         widget_defaults = {
@@ -413,8 +407,6 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
                 "cards_x_nodes_x_widgets"
             ]
         }
-
-        self.assertIn(new_node_id, tile.data)
 
         self.assertEqual(
             tile.data[new_node_id],
@@ -458,13 +450,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             sortorder=0,
         )
 
-        self.assertEqual(
-            tile.data,
-            {
-                str(self.string_node_id): {
-                    "en": {"value": "Default Value", "direction": "ltr"},
-                }
-            },
+        self.assertDictEqual(
+            tile.data[str(self.string_node_id)],
+            {"en": {"value": "Default Value", "direction": "ltr"}},
         )
 
         draft_graph = self.test_graph.create_draft_graph()
@@ -499,11 +487,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
 
         tile.refresh_from_db()
 
-        self.assertEqual(
+        self.assertDictEqual(
             tile.data[str(self.string_node_id)],
-            {
-                "en": {"value": "Overridden Default Value", "direction": "ltr"},
-            },
+            {"en": {"value": "Overridden Default Value", "direction": "ltr"}},
         )
 
     @patch("arches.app.tasks.notify_completion")
@@ -529,13 +515,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             sortorder=0,
         )
 
-        self.assertEqual(
-            tile.data,
-            {
-                str(self.string_node_id): {
-                    "en": {"value": "Default Value", "direction": "ltr"},
-                }
-            },
+        self.assertDictEqual(
+            tile.data[str(self.string_node_id)],
+            {"en": {"value": "Default Value", "direction": "ltr"}},
         )
 
         draft_graph = self.test_graph.create_draft_graph()

--- a/tests/tasks_tests.py
+++ b/tests/tasks_tests.py
@@ -333,8 +333,8 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             nodegroup_id=self.test_graph.get_nodegroups()[0].nodegroupid,
             data={
                 str(self.string_node_id): {
-                    "en": {"value": "test value", "direction": "ltr"}
-                },
+                    "en": {"value": "test value", "direction": "ltr"},
+                }
             },
             sortorder=0,
         )
@@ -343,7 +343,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             tile.data,
             {
                 str(self.string_node_id): {
-                    "en": {"value": "test value", "direction": "ltr"}
+                    "en": {"value": "test value", "direction": "ltr"},
                 }
             },
         )
@@ -353,7 +353,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
         new_node_id = "33333333-3333-3333-3333-333333333333"
         node_to_add = models.Node.objects.create(
             config={
-                "defaultValue": {"en": {"value": "Default Value", "direction": "ltr"}},
+                "defaultValue": {
+                    "en": {"value": "Default Value", "direction": "ltr"},
+                }
             },
             datatype="string",
             description="Added test node",
@@ -373,8 +375,8 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
         models.CardXNodeXWidget.objects.create(
             config={
                 "defaultValue": {
-                    "en": {"value": "Overridden Default Value", "direction": "ltr"}
-                },
+                    "en": {"value": "Overridden Default Value", "direction": "ltr"},
+                }
             },
             card=card,
             node_id=node_to_add.nodeid,
@@ -400,7 +402,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
 
         self.assertEqual(
             tile.data[str(self.string_node_id)],
-            {"en": {"value": "test value", "direction": "ltr"}},
+            {
+                "en": {"value": "test value", "direction": "ltr"},
+            },
         )
 
         widget_defaults = {
@@ -458,7 +462,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             tile.data,
             {
                 str(self.string_node_id): {
-                    "en": {"value": "Default Value", "direction": "ltr"}
+                    "en": {"value": "Default Value", "direction": "ltr"},
                 }
             },
         )
@@ -472,7 +476,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
         ).update(
             config={
                 "defaultValue": {
-                    "en": {"value": "Overridden Default Value", "direction": "ltr"}
+                    "en": {"value": "Overridden Default Value", "direction": "ltr"},
                 },
             }
         )
@@ -497,7 +501,9 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
 
         self.assertEqual(
             tile.data[str(self.string_node_id)],
-            {"en": {"value": "Overridden Default Value", "direction": "ltr"}},
+            {
+                "en": {"value": "Overridden Default Value", "direction": "ltr"},
+            },
         )
 
     @patch("arches.app.tasks.notify_completion")
@@ -517,8 +523,8 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             nodegroup_id=nodegroup_id,
             data={
                 str(self.string_node_id): {
-                    "en": {"value": "Default Value", "direction": "ltr"}
-                },
+                    "en": {"value": "Default Value", "direction": "ltr"},
+                }
             },
             sortorder=0,
         )
@@ -527,7 +533,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             tile.data,
             {
                 str(self.string_node_id): {
-                    "en": {"value": "Default Value", "direction": "ltr"}
+                    "en": {"value": "Default Value", "direction": "ltr"},
                 }
             },
         )

--- a/tests/tasks_tests.py
+++ b/tests/tasks_tests.py
@@ -333,7 +333,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             nodegroup_id=self.test_graph.get_nodegroups()[0].nodegroupid,
             data={
                 str(self.string_node_id): {
-                    "en": {"value": "test value", "direction": "ltr"},
+                    "en": {"value": "test value", "direction": "ltr"}
                 },
             },
             sortorder=0,
@@ -353,9 +353,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
         new_node_id = "33333333-3333-3333-3333-333333333333"
         node_to_add = models.Node.objects.create(
             config={
-                "defaultValue": {
-                    "en": {"value": "Default Value", "direction": "ltr"},
-                },
+                "defaultValue": {"en": {"value": "Default Value", "direction": "ltr"}},
             },
             datatype="string",
             description="Added test node",
@@ -375,7 +373,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
         models.CardXNodeXWidget.objects.create(
             config={
                 "defaultValue": {
-                    "en": {"value": "Overridden Default Value", "direction": "ltr"},
+                    "en": {"value": "Overridden Default Value", "direction": "ltr"}
                 },
             },
             card=card,
@@ -474,7 +472,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
         ).update(
             config={
                 "defaultValue": {
-                    "en": {"value": "Overridden Default Value", "direction": "ltr"},
+                    "en": {"value": "Overridden Default Value", "direction": "ltr"}
                 },
             }
         )
@@ -519,7 +517,7 @@ class UpdateResourceInstanceDataTaskTests(ArchesTestCase):
             nodegroup_id=nodegroup_id,
             data={
                 str(self.string_node_id): {
-                    "en": {"value": "Default Value", "direction": "ltr"},
+                    "en": {"value": "Default Value", "direction": "ltr"}
                 },
             },
             sortorder=0,

